docs/run.md

```diff
@@ -78,6 +78,24 @@ The following ingestors need to run first and in this particular order:
 
 Order doesn't matter for other ingestors.
 
+#### Github Enterprise
```

Comment:
nitpick: GitHub

Reply:
> hey @tdonaworth , thanks so much for adding support for this
> 
> have you been able to successfully test `gitoops` against a self-hosted GH Enterprise? I didn't have anyone in my network who could test this, so would be curious to know if it works alright :)
> 
> I've left a few minor comments regarding this PR.

Yes, I've been running it against a client's GitHub Enterprise instance - I first just hard-coded the values to see if everything else would work; when it did, I refactored what you see here. I'll review the comments and look to make some changes - but Go is my primary language :(